cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -108,11 +114,12 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
     private boolean sslEnabled;
     private boolean authEnabled;
     @SuppressFBWarnings({"UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR"}) // This field is initialized in the fromCrd method
+    protected CruiseControlConfiguration configuration;
```

Comment:
I do not see any changed to how it is initialized. So why does it now need the supression?

Reply:
Needed to add this field so the information could be accessed by non-static  `generateConfigMap` method at the bottom of the class file. Since this field is not initialized in the constructor needed to add the suppression

Reply:
The field was already there. It seems you just moved it and changed its visibility to protected. But I do not see any change to how it is configured. So why does it now need the suppression?